### ISSUE TITLE
OpenApiMixin extensions should include Null values

### ISSUE THREAD

**Issue by loomisdf** [swagger-api/swagger-core#4132]
### Summary
Ran into this issue while working with springdocs-openapi https://github.com/springdoc/springdoc-openapi/issues/1525
Null vendor exensions are allowed via the OpenAPI spec detailed [here](https://swagger.io/docs/specification/openapi-extensions/) 

> The extension value can be a primitive, an array, an object or null

However, they are not being rendered because the OpenApiMixins class is not setup to do so.

### Expected Output
```Json
"x-my-vendor-extensions": {
	"property1": "value1",
        "property2": null
}
```
### Actual Output
```Json
"x-my-vendor-extensions": {
	"property1": "value1"
}
```
### Solution
 I believe the `@JsonInclude(value = Include.ALWAYS)` annotation should be added [here](https://github.com/swagger-api/swagger-core/blob/master/modules/swagger-core/src/main/java/io/swagger/v3/core/jackson/mixin/OpenAPIMixin.java#L14).
```Java
@JsonAnyGetter
@JsonInclude(value = Include.ALWAYS)
public abstract Map<String, Object> getExtensions();
```
A similar workaround has been added to the [springdocs-openapi project](https://github.com/springdoc/springdoc-openapi/commit/caa9b6873a201c31c947ec0153371620aeb26330).

**Comment by ponelat** [comment]
@HugoMario could you take a look into fixing this, if it's still an issue. Thanks!

**Comment by HugoMario** [comment]
fixed by #4468
